Rakefile

```diff
@@ -5,15 +5,6 @@ require 'rubygems/package_task'
 require "rake/testtask"
 require 'psych'
 
-desc "Setup Rubygems dev environment"
-task :setup => ["bundler:checkout"] do
```

Comment:
Could this just call `bundle install` instead of deleting it?

Reply:
Sure, that way we can keep the `bundler:checkout` prerequisite.